packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -510,115 +542,460 @@ test('LP repay', async t => {
+    bridges: { snapshot, since },
+    mint,
+  } = t.context;
+  const cust1 = makeCustomer('Carl', cctp, txPub.publisher, feeConfig);
+
+  const bridgePos = snapshot();
+  const sent1 = await cust1.send(t, 108_000_000n, 'osmo1234advanceHappy');
+  await transmitTransferAck(); // ack IBC transfer for advance
+  // Nothing we can check here, unless we want to inspect calls to `trace`.
+  // `test/exos/advancer.test.ts` covers calls to `log: LogFn` with mocks.
+  // This is still helpful to call, so we can observe "Advance transfer
+  // fulfilled" in the test output.
 
-  // advance sent from PoolSeat to PoolAccount
+  const { calculateAdvance, calculateSplit } = makeFeeTools(feeConfig);
+  const advance = calculateAdvance(usdc.make(sent1.tx.amount));
+  t.log('advancer sent to PoolAccount', advance);
   t.deepEqual(inspectBankBridge().at(-1), {
-    amount: String(expectedAdvance.value),
-    denom: 'ibc/usdconagoric',
+    amount: String(advance.value),
+    denom: uusdcOnAgoric,
     recipient: 'agoric1fakeLCAAddress',
     type: 'VBANK_GIVE',
   });
 
-  // ibc transfer sent over localChain bridge
-  const localBridgeMsg = inspectLocalBridge().at(-1);
-  const ibcTransferMsg = localBridgeMsg.messages[0];
-  t.is(ibcTransferMsg['@type'], '/ibc.applications.transfer.v1.MsgTransfer');
-
-  const expectedReceiver = addressTools.getQueryParams(
-    evidence.aux.recipientAddress,
-  ).EUD;
-  t.is(ibcTransferMsg.receiver, expectedReceiver, 'sent to correct address');
-  t.deepEqual(ibcTransferMsg.token, {
-    amount: String(expectedAdvance.value),
-    denom: 'ibc/usdconagoric',
-  });
+  cust1.checkSent(t, since(bridgePos));
 
-  // TODO #10445 expect PFM memo
-  t.is(ibcTransferMsg.memo, '', 'TODO expecting PFM memo');
+  const emptyMetrics = {
+    encumberedBalance: usdc.makeEmpty(),
+    shareWorth: {
+      numerator: usdc.make(1n),
+      denominator: { value: 1n },
+    },
+    totalBorrows: usdc.makeEmpty(),
+    totalContractFees: usdc.makeEmpty(),
+    totalPoolFees: usdc.makeEmpty(),
+    totalRepays: usdc.makeEmpty(),
+  };
+  const par250 = {
+    numerator: usdc.make(250_000_001n),
+    denominator: { value: 250_000_001n },
+  };
 
-  // TODO #10445 expect routing through noble, not osmosis
-  t.is(
-    ibcTransferMsg.sourceChannel,
-    fetchedChainInfo.agoric.connections['osmosis-1'].transferChannel.channelId,
-    'TODO expecting routing through Noble',
+  t.like(
+    await E(metricsSub)
+      .getUpdateSince()
+      .then(r => r.value),
+    {
+      ...emptyMetrics,
+      encumberedBalance: advance,
+      shareWorth: par250,
+      totalBorrows: advance,
+    },
+    'metrics while advancing',
   );
 
+  await mint(sent1);
+
+  // C8 - "Contract MUST be able to initialize settlement process when Noble mints USDC."
+  // The metrics are a useful proxy, but the contract could lie.
+  // The real test of whether the contract turns minted funds into liquidity is
+  // the ability to advance the funds (in later tests).
```

Comment:
Which later test are you referring to? Should we check the outgoing `localchainBridge` messages in this test and confirm the advance was sent?

Reply:
> Which later test are you referring to?

The racer test, for example.

> Should we check the outgoing localchainBridge messages in this test and confirm the advance was sent?

We did `checkSent()` for the STORY01 advance.

What this comment is saying (or: trying to say) is: after STORY01 completes, the contract _says_ (via metrics) that it still has ~250USDC available. But it could say that it has a zillion USDC available. The proof that it actually has funds available is when it actually transfers them in a future advance.